iceoryx_utils/test/moduletests/test_atomic_relocatable_ptr.cpp

```diff
@@ -156,19 +156,20 @@ TEST_F(AtomicRelocatablePointer_test, dereferenceOperator)
 // copied location
 TEST_F(AtomicRelocatablePointer_test, memoryRelocation)
 {
-    Memory<1024> memory;
+    constexpr uint64_t ALIGNMENT_OF_PTR {alignof(Ptr<byte_t>)};
+    Memory<1024, ALIGNMENT_OF_PTR> memory;
     memory[1000] = 37;
```

Comment:
The data is just one byte, so there is no problem to construct it at the byte 1000.